src/index.ts

```diff
@@ -33,39 +35,38 @@ export interface CrawlResult {
 }
 
 interface CrawlOptions {
-  url: string;
+  url: url;
   crawl: boolean;
   results: LinkResult[];
   cache: Set<string>;
   checkOptions: CheckOptions;
 }
 
-/**
- * Instance class used to perform a crawl job.
- */
+const isHtml = (response: gaxios.GaxiosResponse) => {
```

Comment:
Why move this to an arrow function?  Regular functions are honestly fine unless you're dealing with `this` pointer hell

Reply:
No good reason. I just prefer arrows. A nit.